packages/client-python/tests/RocketRideClient_test.py

```
@@ -2240,6 +2240,154 @@ def test_get_websocket_uri_normalization(input_uri: str, expected_uri: str) -> N
 
 
+# ============================================================================
+# File Store Operations
+# ============================================================================
+
+
+class TestFileStoreOperations:
+    """Test handle-based file store operations against a live server."""
+
+    def _unique_path(self, name: str = 'test') -> str:
+        return f'.test-store/py-{name}-{"".join(random.choices(string.ascii_lowercase, k=8))}'
```

Ruff (0.15.7)

[error] Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)